src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/cas2/ApplicationsController.kt

```diff
@@ -39,16 +40,20 @@ class ApplicationsController(
   private val userService: NomisUserService,
 ) : ApplicationsCas2Delegate {
 
-  override fun applicationsGet(isSubmitted: Boolean?): ResponseEntity<List<ApplicationSummary>> {
+  override fun applicationsGet(isSubmitted: Boolean?, page: Int?): ResponseEntity<List<ApplicationSummary>> {
     val user = userService.getUserForRequest()
 
-    val applications = when (isSubmitted) {
-      true -> applicationService.getSubmittedApplicationsForUser(user)
-      false -> applicationService.getUnsubmittedApplicationsForUser(user)
-      null -> applicationService.getAllApplicationsForUser(user)
+    val pageCriteria = PageCriteria("created_at", SortDirection.desc, page)
```

Comment:
Changing the sort order from `desc` to `asc` results in test `Get all applications with pagination returns 200 with correct body and header` still passing.  Bug [CAS2-374](https://dsdmoj.atlassian.net/browse/CAS2-374) raised in backlog.